src/main/scala/io/iohk/ethereum/jsonrpc/server/http/JsonRpcHttpServer.scala

```diff
@@ -54,37 +54,39 @@ trait JsonRpcHttpServer extends Json4sSupport with RateLimit with Logger {
       }
       .result()
 
+  val rateLimit = new RateLimit(config.rateLimit)
+
   val route: Route = cors(corsSettings) {
     (path("healthcheck") & pathEndOrSingleSlash & get) {
       handleHealthcheck()
     } ~ (path("buildinfo") & pathEndOrSingleSlash & get) {
       handleBuildInfo()
     } ~ (pathEndOrSingleSlash & post) {
-      (extractClientIP & entity(as[JsonRpcRequest])) { (clientAddress, request) =>
-        handleRequest(clientAddress, request)
-      } ~ entity(as[Seq[JsonRpcRequest]]) { request =>
-        handleBatchRequest(request)
+      // TODO: maybe rate-limit this one too?
+      entity(as[JsonRpcRequest]) {
+        case statusReq if statusReq.method == FaucetJsonRpcController.Status =>
```

Comment:
Here we can coordinate with devops. And change the endpoint used for the status health check. 
Today we are validating the status “healthcheck” through the endpoint json rpc “faucet_status”. Here we can change it for the request [GET] http “healthcheck”. This endpoint internally, works the same. 

In this way, we can remove the following condition:
if statusReq.method == FaucetJsonRpcController.Status

In this point we need coordinate with devops and change the endpoint that we have in the following line:
https://github.com/input-output-hk/mantis-ops/blob/784f14f0218ef9f383cd57d96367d6e6a07b5ea7/docker/mantis-faucet.nix#L8

What do you think?

Reply:
I'm looking forward but we should be aware if someone other than us is using this endpoint for health checking...
If it is not, i'm sure I can remove this workaround. WDYT, @dranik-blox?

Reply:
Maybe we can create another task about it. 
What do you think?

Reply:
https://jira.iohk.io/browse/ETCM-510